siliconcompiler/core.py

```diff
@@ -334,10 +334,10 @@ def create_cmdline(self, progname, description=None, switchlist=[]):
                 self.set('mode', cmdargs['mode'], clobber=True)
             if 'techarg' in cmdargs.keys():
                 print("NOT IMPLEMENTED")
-                sys.exit()
+                raise NotImplementedError('NOT IMPLEMENTED')
```

Comment:
Shouldn't this be `techarg not implemented`

Reply:
Changed these to, "NOT IMPLEMENTED: '[param]' parameter"